### DISCUSSION TITLE
SPI CAN bus supported?

### DISCUSSION THREAD

**Discussion by mrspirytus** [OE4T/meta-tegra#517] [Q&A]
Hello Everyone

I was wondering if anyone is using can CAN bus devices in this project. I found this [CAN-BUS-FD](https://www.seeedstudio.com/CAN-BUS-FD-HAT-for-Raspberry-Pi-p-4742.html#) unit I am thinking of getting a few to test.
It is for Raspberry Pi, and the FD version also supports Jetson Nano. It has two MCP2518FD

And sources and installation are [here](https://wiki.seeedstudio.com/2-Channel-CAN-BUS-FD-Shield-for-Raspberry-Pi/)

**Comment by dwalkes** [comment]
Sorry that this hasn't had any answers, wish I could help.  If you've made progress since posting though I'd be interested to hear about it.
